types/react-native-communications/index.d.ts

```diff
@@ -1,11 +1,12 @@
 // Type definitions for react-native-communications 2.2
 // Project: https://github.com/anarchicknight/react-native-communications
 // Definitions by: HuHuanming <https://github.com/huhuanming>
+//                 Paito <https://github.com/PaitoAnderson>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
 export function phonecall(phoneNumber: string, isPrompt: boolean): void;
-export function email(to: string, cc: string, bcc: string, subject: string, body: string): void;
+export function email(to: string[] | null, cc: string[]| null, bcc: string[] | null, subject: string | null, body: string | null): void;
```

Comment:
Add an overload which supports 0 arguments, since according to the docs that's valid. (and add a test)

Reply:
Good catch, done!